sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -45,17 +48,29 @@ export interface KeyVaultKey {
    * Operations allowed on this key
    */
   keyOperations?: JsonWebKeyOperation[];
+  /**
+   * @member {KeyProperties} [properties] The properties of the key.
+   */
+  properties: KeyProperties;
 }
 
 /**
  * @interface
  * An interface representing the Properties of a key
  */
-export interface KeyProperties extends ParsedKeyVaultEntityIdentifier {
+export interface KeyProperties {
```

Comment:
Missing id here as well.

Reply:
See https://apiview.azurewebsites.net/Assemblies/Review/d74052731cb94759804beaa492b7a17a for an updated canonical example.

Reply:
Fixing it